### ISSUE TITLE
obsidian festival

### ISSUE THREAD

**Issue by victorr44** [shishkabob27/CardWars#25]
yesss and obsidian festival where u can get black cards in gold chest

_Originally posted by @Sabersabersaber123 in https://github.com/shishkabob27/CardWars/issues/23#issuecomment-1837462103_

**Comment by Sabersabersaber123** [comment]
yes also all the different card wars carnival events in the algebraic chest, like theres events with chances to pull heros, more gold cards, and black cards like obsidian carnival
info here:
https://cardwars.fandom.com/wiki/Card_Wars_Carnival
